### PR TITLE
feat(Misc): Enable some cool modules

### PR DESCRIPTION
**Pull request by e0ipso**
Task:

* [x] Ready for review
* [x] Ready for merge

Enables the `snail` module and the `admin_login_path` (companion to `material_admin`).

### CODE REVIEW

**Comment by dawehner** [comment]
0) Totally +1
a) Travis broke somehow on 5.6, any idea?
b) I think we should try to push for alphas at least on all the used modules

File: composer.json [hunk 1]
```diff
@@ -65,7 +65,9 @@
         "drupal/materialize": "1.x-dev",
         "drupal/schemata": "^1.0@alpha",
         "drupal/openapi": "^1.0@alpha",
-        "drupal/material_admin": "1.x-dev"
+        "drupal/material_admin": "1.x-dev",
+        "drupal/snail": "1.x-dev",
```
Comment (dawehner) [inline composer.json:69]: Note: There is an alpha for the snail module now :)

Comment (cjgratacos) [reply composer.json:69]: @dawehner Travis failed on 5.6 on PHP out of memory exception caused by composer. If the latest 8.x-1.x gets merge into this branch, that error shouldn't happen :)

**Comment by dawehner** [comment]
Let's go with it for now.

**Pull request merged**